readalongs/align.py

```diff
@@ -179,6 +217,13 @@ def frames_to_time(frames):
     for seg in ps.seg():
         start = frames_to_time(seg.start_frame)
         end = frames_to_time(seg.end_frame + 1)
+        if do_not_align_segments and method == 'remove':
+            start += (calculate_adjustment(start,
+                                           do_not_align_segments) / 1000)
+            end += (calculate_adjustment(end, do_not_align_segments) / 1000)
+            start, end = correct_adjustments(start, end, do_not_align_segments)
+            start /= 1000
+            end /= 1000
```

Comment:
I don't like this mucking around with seconds and ms. Can we pick a consistent unit to use everywhere, and only convert (if necessary) when data is read from or written to file? Ideally, I would prefer having this function here, as well as calculate_adjustment and correct_adjustment all work in the same unit, probably the ms.

Reply:
OK, I refactored it (18b713d) so that `calculate_adjustment` and `correct_adjustments` are both getting passed timestamps in ms. Let me know if you think my change is an improvement.